generated/nifake/nifake/session.py

```diff
@@ -1295,7 +1297,7 @@ def import_attribute_configuration_buffer(self, configuration):
         '''
         vi_ctype = _visatype.ViSession(self._vi)  # case S110
         size_in_bytes_ctype = _visatype.ViInt32(0 if configuration is None else len(configuration))  # case S160
-        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_import_buffer_to_array(configuration), library_type=_visatype.ViInt8)  # case B520
+        configuration_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_to_bytes(configuration), library_type=_visatype.ViInt8)  # case B520
```

Comment:
In all the other drivers, export_attribute_configuration changes but not in nifake. Are we missing something?